jobs/build.Jenkinsfile

```diff
@@ -418,17 +418,19 @@ lock(resource: "build-${params.STREAM}") {
                 }
             }
 
-            // parallel build these artifacts
-            def pbuilds = [:]
-            ["Aliyun", "AWS", "Azure", "AzureStack", "DigitalOcean", "Exoscale", "GCP", "IBMCloud", "Nutanix", "OpenStack", "VirtualBox", "VMware", "Vultr"].each {
-                pbuilds[it] = {
+            // parallel build these artifacts over two runs
```

Comment:
The first question I have in the future when I read this code is "why?". Why split it over two runs when no steps depend on each other?

We know the answer now.. Let's add a comment explaining?